ubuntu_20.04-stable.dockerfile

```diff
@@ -9,7 +9,7 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 # install apt dependencies
 RUN apt-get -y  update
 RUN apt-get install -y software-properties-common \
-                       python3-pip \
+                       python-is-python3 \
```

Comment:
Is this necessary? Running update-alternatives I think is the safer way to do this.

Reply:
AS we have already `python3` and `pip3` installed, I just wanted to create a symlink to avoid the additional installational process.